resources/util.sh

```diff
@@ -120,6 +120,7 @@ function startMysql() {
   echo "Starting mysql..."
   setDoguLogLevel
   doguctl state "ready"
+  doguctl config --rm "local_state"
```

Comment:
We can leave it in the code as a security measure, but if I am not mistaken this should never do anything, right?  Because it's not set on a normal start up and on the upgrade path it will be deleted by the post-upgrade.sh first.

Reply:
Yeah, I started adding it because dogus sometimes write error messages there which should be discarded after normal startup.